test/test_graphene.py

```diff
@@ -5,11 +5,84 @@
 import numpy as np
 import shutil
 import pytest
+import os
+from scipy import sparse 
 
 tempdir = tempfile.mkdtemp()
 TEST_PATH = "file:/{}".format(tempdir)
 TEST_DATASET_NAME = "testvol"
+MESH_TEST_DATASET_NAME = "meshvol"
 PCG_LOCATION = "http://localhost/segmentation/1.0/"
+PCG_MESH_LOCATION = "http://localhost./meshing/1.0/"
```

Comment:
localhost `.` /meshing?

Reply:
i have no idea about this.. when i took this off it complained that my mock wasn't formatted correctly...

Reply:
I figured it out.. was how the meshing url was constructed... fixed now